mods/e2140/content/ed/infantry/a03/rules.yaml

```
@@ -0,0 +1,26 @@
+ed_infantry_a03:
+	Inherits@1: ^EdInfantry
+	Inherits@2: ^CoreArmedSelfInfantry
+	Tooltip:
+		Name: A03 Android
+	Valued:
+		Cost: 100
+	Buildable:
+		IconPalette:
+		Queue: Infantry.ED
+	Health:
+		HP: 28
+	RevealsShroud:
+		Range: 2c896
+	Armament@PRIMARY:
+		Weapon: ed_infantry_a03
+		LocalOffset: 400,0,50
+		MuzzlePalette:
+	EnhancedEncyclopedia:
+		Category: ED - Infantry
+		Order: 3
+		Animation: DATABASE/A03.FLC
+		Title: A03 Android
```

Use original name *Android A03*:

```suggestion
		Title: Android A03
```